views/pulsar/v2/helpers/nav.html.twig

```diff
@@ -56,12 +56,12 @@
 {% endmacro %}
 
 
-{% macro nav_item(options) %}
+{% macro nav_item(options, index) %}
 {% spaceless %}
     {% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
     {% import '@pulsar/pulsar/v2/helpers/util.html.twig' as util %}
 
-    <li
+    <li id="nav-item-{{ index }}"
```

Comment:
Sounds good in that case, sorry by name spacing nav instances I meant... have the `<li>`s id something like `id="nav-{{ navInstanceIndex}}-item-{{ itemIndex}}` to get around multiple instances - but if multiple instances are not a concern, ignore me!